chromadb/__init__.py

```diff
@@ -255,3 +282,9 @@ def AdminClient(settings: Settings = Settings()) -> AdminAPI:
 
     """
     return AdminClientCreator(settings=settings)
+
+
+def _stringify_headers(headers: Optional[Dict[str, str]]) -> None:
```

Comment:
I feel like we can be more suggestive with the type-hinting here. If `headers` is already `Dict[str, str]` then we would not need to cast. More explicitly,
```suggestion
def _stringify_headers(headers: Optional[Dict[str, str | int]]) -> None:
```

I would have suggested a `SupportsStr` method but I think every(?) object in python has a `__str__` method? see related exchange https://www.mail-archive.com/python-ideas@python.org/msg24545.html